src/sentry/eventstream/base.py

```diff
@@ -22,6 +32,15 @@ class ForwarderNotRequired(NotImplementedError):
     """
 
 
+class GroupState(TypedDict):
+    is_new: bool
+    is_regression: bool
+    is_new_group_environment: bool
```

Comment:
I think we need `primary_hash` here as well. We might not include this for perf issues, but I want to send error issues through here eventually as well.